### PR TITLE
install R for MSstats

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

**Comment by timosachsenberg** [comment]
works fine on my PC with R 3.5 and the current bioconductor version.
Maybe we need to fix that to a known version so scripts don't break if bioconductor / Msstats is update?

**Comment by mwalzer** [comment]
That is probably a good idea. Do you think the PPA is necessary?

Would this work, too?
```
apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
add-apt-repository 'deb [arch=amd64,i386] https://cran.rstudio.com/bin/linux/ubuntu xenial/'
apt-get update
```

**Comment by timosachsenberg** [comment]
Hmm I don’t know if this allows to choose a specific r version. I would like to fix it to 3.5 and a specific bioconductor Version as an update might break the msstats package

**Comment by mwalzer** [comment]
Yes, I agree that it is a good idea to fix the versions. Do you know how to fix versions for bioclite installations?

File: batteries-included/Dockerfile [hunk 1]
```diff
@@ -62,7 +62,13 @@ RUN apt-get install -y --no-install-recommends --no-install-suggests libsvm-dev
 													seqan-dev \
 													libbz2-dev \
 													coinor-libcoinmp-dev
-
+# install R 3.5
+RUN apt-get install -y software-properties-common
```
Comment (mwalzer) [inline batteries-included/Dockerfile]:
```suggestion
RUN apt-get install -y software-properties-common apt-transport-https
RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
RUN add-apt-repository 'deb [arch=amd64,i386] https://cloud.r-project.org/bin/linux/ubuntu xenial-cran35/'
RUN apt-get update
RUN apt install -y r-base
```

File: batteries-included/Dockerfile [hunk 2]
```diff
@@ -62,7 +62,13 @@ RUN apt-get install -y --no-install-recommends --no-install-suggests libsvm-dev
...
+RUN add-apt-repository -y ppa:marutter/c2d4u3.5
```
Comment (mwalzer) [inline batteries-included/Dockerfile]:
```suggestion

```

File: batteries-included/Dockerfile [hunk 3]
```diff
@@ -62,7 +62,13 @@ RUN apt-get install -y --no-install-recommends --no-install-suggests libsvm-dev
...
+RUN apt-get update
```
Comment (mwalzer) [inline batteries-included/Dockerfile:69]:
```suggestion

```

File: batteries-included/Dockerfile [hunk 4]
```diff
@@ -62,7 +62,13 @@ RUN apt-get install -y --no-install-recommends --no-install-suggests libsvm-dev
...
+RUN apt install -y r-api-3.5
```
Comment (mwalzer) [inline batteries-included/Dockerfile]:
```suggestion

```

**Review by mwalzer — CHANGES_REQUESTED** [review]
I really like the R integration, just added suggestions how to use the official cran repo (see here: https://cran.r-project.org/bin/linux/ubuntu/)

**Review by mwalzer — APPROVED** [review]

**Pull request merged**